Jenkinsfile

```diff
@@ -138,18 +132,25 @@ pipeline {
                 mkdir -p $COUCHDB_IO_LOG_DIR
 
                 # Build CouchDB from tarball & test
-                builddir=$(mktemp -d)
-                cd $builddir
+                mkdir build
+                cd build
                 tar -xf $WORKSPACE/apache-couchdb-*.tar.gz
                 cd apache-couchdb-*
                 ./configure --with-curl
                 gmake check || (build-aux/logfile-uploader.py && false)
 
                 # No package build for FreeBSD at this time
-                rm -rf $builddir $COUCHDB_IO_LOG_DIR
```

Comment:
FreeBSD isn't running inside a container, and you didn't add a separate workspace here. Without using a `/tmp` directory, and without cleaning up the builddir, the next build will have files remaining in the folder. Are you 100% sure this isn't an issue on multiple successive builds? We can always restore it if necessary, I guess.

Reply:
My thought was the `deleteDir()` at the beginning should take care of things for us, but I'll admit I didn't specifically check that it was doing what it said.